src/mouse_handler_base.cpp

```diff
@@ -146,27 +146,31 @@ void mouse_handler_base::mouse_press(const SDL_MouseButtonEvent& event, const bo
 	map_location loc = gui().hex_clicked_on(event.x,event.y);
 	mouse_update(browse, loc);
 	
-	static time_t touch_timestamp = 0;
-	using std::chrono::duration_cast;
-	using std::chrono::microseconds;
-	using std::chrono::high_resolution_clock;
-	high_resolution_clock::time_point touch_timestamp_cpp;
+	static clock_t touch_timestamp = 0;
+//	using std::chrono::duration_cast;
+//	using std::chrono::duration;
+//	using std::chrono::milliseconds;
+//	using std::chrono::high_resolution_clock;
+//	high_resolution_clock::time_point touch_timestamp_cpp;
 	
 	if (is_touch_click(event)) {
 		if (event.state == SDL_PRESSED) {
 			cancel_dragging();
-			touch_timestamp = time(NULL);
-			touch_timestamp_cpp = high_resolution_clock::now();
+			touch_timestamp = clock();
```

Comment:
`clock()` works for me on iOS and Mac. There is a clock() call in Lua sources (though, Lua only relies on it for randomness source). It's C89 and POSIX-2001 (https://linux.die.net/man/3/clock)... Hope it's enough to verify portability, until we can actually test it everywhere.